bvm/ballerina-core/src/main/java/org/ballerinalang/BLangProgramRunner.java

```diff
@@ -88,6 +94,15 @@ public static void runMain(ProgramFile programFile, String[] args) {
         }
     }
 
+    private static void initializeDistributeTransactionStatus(ProgramFile programFile) {
+        boolean distributedTxEnabled = false; //TODO:Default will be true. Read from new VMOptions
+        String distributedTxEnabledProp = System.getProperty(Constants.DISTRIBUTED_TRANSACTION_ENABLED);
+        if (distributedTxEnabledProp != null && !distributedTxEnabledProp.isEmpty()) {
```

Comment:
If you're checking to see if it's not empty also .. that means .. we expect a value from it .. such as "true" or "false"? .. if that's the case, we have to boolean parse that value and set it. In this way, any value put into the system property, the flag will be true.